docs/1-Using-Fleet/standard-query-library/standard-query-library.yml

```diff
@@ -51,6 +51,16 @@ spec:
 ---
 apiVersion: v1
 kind: query
+spec:
+  name: Get authorized keys for Domain Joined Accounts
+  platforms: macOS, Linux
+  description: List authorized_keys for each user on the system.
+  query: SELECT * FROM users CROSS JOIN authorized_keys USING (uid) and username in (SELECT distinct(username) FROM last);
```

Comment:
When I attempt to run this query, as a live query, in the Fleet UI, I receive the following error: `Syntax error found near Identifier (Join USING Clause)`

I'm relatively new to SQL so I apologize that I can't provide a quick fix.